src/coredump/coredumpctl.c

```diff
@@ -207,6 +211,14 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_no_legend = true;
                         break;
 
+                case ARG_DEBUGGER:
+                        if (arg_debugger) {
+                                log_error("cannot use --debugger more than once");
```

Comment:
Doesn't matter much, but we prefer proper english sentences as log messages, i.e. with an uppercase first letter, and a full stop at the end...

Also, we try to suffix all cmdline options that take a parameter with `=` in our docs and messages to indicate that they expect an option, would be good to follow this here too

Reply:
Actually this check should be dropped.